website/docs/labs/capsule-exec.md

```diff
@@ -0,0 +1,162 @@
+---
+id: capsule-exec
+title: Exec syscall in Capsule
+---
+
+## Introduction
+
+[Exec](https://github.com/nervosnetwork/rfcs/blob/master/rfcs/0034-vm-syscalls-2/0034-vm-syscalls-2.md#exec) is a new syscall provided by [ckb2021](https://github.com/nervosnetwork/rfcs/blob/master/rfcs/0037-ckb2021/0037-ckb2021.md). To understand what exec syscall does, we recommend you read this article first: [https://man7.org/linux/man-pages/man3/exec.3.html](https://man7.org/linux/man-pages/man3/exec.3.html). In short: Exec runs an executable file from specified cell data in the context of an already existing vm, replacing the previous executable. The used cycles does not change, but the code, registers and memory of the vm are replaced by those of the new program. This means, **control flow will never return to the main script**.
```

Comment:
[Exec](https://github.com/nervosnetwork/rfcs/blob/master/rfcs/0034-vm-syscalls-2/0034-vm-syscalls-2.md#exec) is a new syscall provided by [ckb2021](https://github.com/nervosnetwork/rfcs/blob/master/rfcs/0037-ckb2021/0037-ckb2021.md). To understand what exec syscall does, we recommend reading [this article] (https://man7.org/linux/man-pages/man3/exec.3.html) first. In short: Exec runs an executable file from specified cell data in the context of an already existing VM, replacing the previous executable. The used cycles do not change, but the code, registers, and memory of the VM are replaced by those of the new program, meaning **control flow will never return to the main script**.